traffic_ops/traffic_ops_golang/crconfig/config_test.go

```diff
@@ -26,13 +26,14 @@ import (
 	"testing"
 	"time"
 
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/test"
 	"gopkg.in/DATA-DOG/go-sqlmock.v1"
```

Comment:
By convention we generally leave a blank line between project-internal imports and external packages